src/model/article-vor.v6.yaml

```diff
@@ -65,6 +65,24 @@ allOf:
             items:
                 $ref: ../misc/blocks-all.v2.yaml
             minItems: 1
+        editorEvaluation:
+            type: object
+            properties:
+                id:
+                    $ref: ../misc/html-id.v1.yaml
+                doi:
+                    $ref: ../misc/doi.v1.yaml
```

Comment:
looks like: `10.7554/eLife.62927.sa0`

@gnott I remember years ago a decision to stop using sub-dois - was that limited to just abstracts or figures or something similar?

Reply:
You remember correctly there was a change years ago, specifically the sub-dois were being reported to Crossref as component DOIs. What remained for more recent articles was a sub-dois for each of the `<sub-article>` XML tags -- the decision letter, and author response, and now is to include an editor's evaluation -- and these are reported to Crossref as peer review materials. The unique DOI allows each of them to be registered as such and can link back to the main article's DOI.